custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -170,7 +170,7 @@ def get_prompt_column_name(self):
             deployment = dr.Deployment.get(self._params["deployment_id"])
             return deployment.model["prompt"]
         except Exception:
-            logger.exception(
+            logger.info(
```

Comment:
However this will hide the errors. Maybe not something user would want to see if this unexpected, but also will make troubleshooting harder.

Reply:
How about I make it a warning? I have observed that prompt column name is not always set on deployments and things work out as expected anyway

Reply:
I dont mean leaving it an exception. I mean losing any stack trace completely. If its a stack trace with warning its still much better than nothing.

Reply:
So a logger.warning(exc_info=True) would show the failure

Reply:
Done